### DISCUSSION TITLE
Support Clarity

### DISCUSSION THREAD

**Discussion by Devendra2104** [vladimir-bukhtoyarov/bucket4j#403] [General]
are they suporting jav8 and springboot 2.4.2 version?

**Comment by vladimir-bukhtoyarov** [comment]
Java 8 is still being supported via dedicated articfacts with "_jdk8" suffix. For example:
* https://mvnrepository.com/artifact/com.bucket4j/bucket4j_jdk8-core/8.3.0
* https://mvnrepository.com/artifact/com.bucket4j/bucket4j_jdk8-jcache/8.3.0

Bucket4j does not provide integration for springboot,so you are free to use any springboot's version.
